clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -3274,6 +3279,27 @@ static void handleIntelReqdSubGroupSize(Sema &S, Decl *D,
   S.AddIntelReqdSubGroupSize(D, AL, E);
 }
 
+IntelNamedSubGroupSizeAttr *
+Sema::MergeIntelNamedSubGroupSizeAttr(Decl *D,
+                                      const IntelNamedSubGroupSizeAttr &A) {
+  if (checkAttrMutualExclusion<IntelNamedSubGroupSizeAttr>(*this, D, A))
```

Comment:
```suggestion
  if (checkAttrMutualExclusion<IntelReqdSubGroupSizeAttr>(*this, D, A))
```
We may be lacking test coverage. ;-)

Reply:
Yikes!